src/js/player.js

```diff
@@ -3599,6 +3599,11 @@ class Player extends Component {
     if (this.tech_) {
       this.tech_.clearTracks('text');
     }
+
+    this.removeClass('vjs-playing');
+    this.removeClass('vjs-has-started');
```

Comment:
the others make sense but I don't think we want to remove this one. Yes, the player is reset, but it should still be in started state, could be a breaking change, potentially as well.

Reply:
@gkatsev thank you for taking the time to review this PR and for the feedback. I removed `this.removeClass('vjs-has-started');`